### PR TITLE
LC24 minutes

### PR DESCRIPTION
**Pull request by thomas-fossati**
Note that the "spec works" presentation is not minuted.  I don't remember who was in charge though...

### CODE REVIEW

File: lc24/README.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+|--|--|--|--|
+| Arnaud de Grandmaison | Arm | FR | Onsite |
+| Arto Niemi | Huawei | FI | Onsite
+| Carsten Weinhold | Barkhausen Institut | DE | Remote |
+| Guilhem Bryant | Arm | GB | Remote |
+| Ionut Mihalcea | Arm | GB | Onsite |
+| Jimmy Kjällman | Ericsson | FI | Onsite
+| Leonardo Garcia | Linaro | BR | Onsite |
+| Muhammad Usama Sardar | TU Dresden | DE | Onsite |
+| Ondrej Smid | Ericsson | SE | Onsite
```
Comment (thomas-fossati) [inline lc24/README.md]:
```suggestion
| Arto Niemi | Huawei | FI | Onsite |
| Carsten Weinhold | Barkhausen Institut | DE | Remote |
| Guilhem Bryant | Arm | GB | Remote |
| Ionut Mihalcea | Arm | GB | Onsite |
| Jimmy Kjällman | Ericsson | FI | Onsite
| Leonardo Garcia | Linaro | BR | Onsite |
| Muhammad Usama Sardar | TU Dresden | DE | Onsite |
| Ondrej Smid | Ericsson | SE | Onsite |
```

**Review by paulhowardarm — APPROVED** [review]
Nicely curated! LGTM.

File: lc24/README.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+
+[link to slides](https://github.com/tls-attestation/materials/blob/main/lc24/standardisation-recap.pdf)
+
+### Challenges in the formal verification of attested TLS
+
+[link to slides](https://github.com/tls-attestation/materials/blob/main/lc24/Formalization-Usama.pdf)
+
+- Challenges in specification 
+    - Incomplete and outdated specs for Intel's RA-TLS
+- Chellenges from formal perspective
```
Comment (muhammad-usama-sardar) [inline lc24/README.md]:
```suggestion
- Challenges from formal perspective 
```

File: lc24/README.md [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+
+[link to slides](https://github.com/tls-attestation/materials/blob/main/lc24/Formalization-Usama.pdf)
+
+- Challenges in specification 
+    - Incomplete and outdated specs for Intel's RA-TLS
+- Chellenges from formal perspective 
+    - Very few comments in Inria's formal model
+    - Incomplete validation of draft 20 artifacts
+    - Not easily extensible
+- [link to slides](https://github.com/tls-attestation/materials/blob/main/lc24/Formalization-Usama.pdf)
```
Comment (muhammad-usama-sardar) [inline lc24/README.md]:
```suggestion
```

remove this as link already present at line 164

File: lc24/README.md [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+The source of the X509 chain is the BoringSSL API `SSL_peer_get_full_cert_chain()` [here](https://commondatastorage.googleapis.com/chromium-boringssl-docs/ssl.h.html#SSL_get_peer_full_cert_chain).
+
+This API just returns `NULL` if the peer is using a non-X509 credential, so there's no way with BoringSSL to get other types of credential as raw bytes.
+
+Hence probably needs swapping out BoringSSL and bringing in OpenSSL and making use of the work of Carsten's team.
+
+## Spec work
+
+Triaged existing issues, created new issues, assigned issues to the [Vancouver milestone](https://github.com/tls-attestation/draft-tls-attestation/issues?q=is%3Aopen+is%3Aissue+milestone%3A%22Vancouver+%28was%3A+Brisbane%29%22).
+
```
Comment (muhammad-usama-sardar) [inline lc24/README.md:215]:
```suggestion
## FV work

- discussed open issues with participants
- discussed the next steps for the paper on formal verification
```

**Review by muhammad-usama-sardar — APPROVED** [review]
a few comments

**Pull request merged**